crypto/x509v3/v3_tlsf.c

```diff
@@ -1,14 +1,15 @@
 /*
- * Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2015-2017 The OpenSSL Project Authors. All Rights Reserved.
  *
  * Licensed under the OpenSSL license (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
 
-#include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Flip these two. Trouble is that internal/cryptlib.h includes a whole bunch of headers and we can't be sure that none of them ends up triggering "position sensitivity" on Windows. Well, one can argue that it's OK since it works(\*), but it's not inappropriate to follow same pattern everywhere.

(\*) after all "position sensitivity" affects which declarations you "see" and if you don't rely on what you don't see, then it's not a problem;